src/menu.c

```diff
@@ -6569,8 +6591,11 @@ void set_language(s32 language) {
     func_8006ECE0();
 }
 
+/**
+ * Returns TRUE if the player has adventure two unlocked.
+ */
 s32 is_adventure_two_unlocked(void) {
-    return D_80126448 & 1;
+    return D_80126448 & TRUE;
```

Comment:
I think in this case TRUE, is an incidental define match is it not? Maybe we could create an or set of defines for the different bits for this value?

Reply:
Yep that's an IQ fade on my end, shall have an enum for both sets of flags at some point